src/js/components/stock-movement-wizard/StockMovementVerifyRequest.jsx

```diff
@@ -0,0 +1,338 @@
+      newName.replace(/ /gi, '');
+    }
+    return `${values.movementNumber} - ${newName}`;
+  }
+
+  getAdditionalWizardTitle() {
+    const { currentPage, values } = this.state;
+    const shipped = values.shipped ? 'SHIPPED' : '';
+    const received = values.received ? 'RECEIVED' : '';
+    if ((this.props.hasPackingSupport && currentPage === 6) ||
+      (!this.props.hasPackingSupport && currentPage === 5)) {
+      return (
+        <span className="shipment-status float-right">
+          {`${shipped || received || 'PENDING'}`}
+        </span>
+      );
+    }
+    return null;
+  }
+
+  updateWizardValues(currentPage, values) {
+    this.setState({ currentPage, values });
+    if (values.movementNumber && (values.id || values.stockMovementId)) {
+      const {
+        actionLabel, defaultActionLabel, actionUrl, listLabel, defaultListLabel, listUrl,
+      } = this.props.breadcrumbsConfig;
+      this.props.updateBreadcrumbs([
+        { label: listLabel, defaultLabel: defaultListLabel, url: listUrl },
+        { label: actionLabel, defaultLabel: defaultActionLabel, url: actionUrl },
+        { label: values.movementNumber, url: actionUrl, id: values.id || values.stockMovementId },
+      ]);
+    }
+  }
+
+  dataFetched = false;
+
+  /**
+   * Fetches initial values from API.
+   * @public
+   */
+  fetchInitialValues() {
+    if (this.props.match.params.stockMovementId) {
+      this.props.showSpinner();
+      const url = `/openboxes/api/stockMovements/${this.props.match.params.stockMovementId}`;
+      apiClient.get(url)
+        .then((response) => {
+          const resp = response.data.data;
+          const originType = resp.origin.locationType;
+          const destinationType = resp.destination.locationType;
+          const values = {
+            ...resp,
+            stockMovementId: resp.id,
+            movementNumber: resp.identifier,
+            origin: {
+              id: resp.origin.id,
+              type: originType ? originType.locationTypeCode : null,
+              name: resp.origin.name,
+              label: `${resp.origin.name} [${originType ? originType.description : null}]`,
+            },
+            destination: {
+              id: resp.destination.id,
+              type: destinationType ? destinationType.locationTypeCode : null,
+              name: resp.destination.name,
+              label: `${resp.destination.name} [${destinationType ? destinationType.description : null}]`,
+            },
+            requestedBy: {
+              id: resp.requestedBy.id,
+              name: resp.requestedBy.name,
+              label: resp.requestedBy.name,
+            },
+            requestType: {
+              name: resp.requestType.name,
+              label: resp.requestType.name,
+            },
+          };
+
+          let currentPage = 1;
+          switch (values.statusCode) {
+            case 'NEW':
```

Comment:
There is no way to get "NEW" or "CREATED" status codes in this workflow.

Reply:
done